src/macaron/config/defaults.ini

```diff
@@ -43,6 +43,12 @@ dep_tool_gradle = cyclonedx-gradle:1.7.4
 timeout = 2400
 recursive = False
 
+# This is the repo finder script.
+[repofinder]
```

Comment:
Since this is only going to target Java projects, can you please rename it to `repofiner.java` and adjust the comment above?